schema.yaml

```diff
@@ -64,17 +64,17 @@ mapping:
                         type: str
                         required: true
                         enum:
-                            - c1.c28m225
-                            - c1.c28m475
-                            - c1.c36m100
-                            - c1.c36m225
-                            - c1.c36m900
-                            - c1.c36m975
-                            - c1.c60m1975
-                            - c1.c125m225
-                            - c1.c125m425
-                            - g1.c36m100g1
-                            - g1.c7m20g1
+                            - c1.c28m225d50
+                            - c1.c28m475d50
+                            - c1.c36m100d50
+                            - c1.c36m225d50
+                            - c1.c36m900d50
+                            - c1.c36m975d50
+                            - c1.c60m1975d50
+                            - c1.c125m225d50
+                            - c1.c125m425d50
+                            - g1.c36m100g1d50
+                            - g1.c7m20g1d50
```

Comment:
@bgruening Search in your email client for the job id _"11ac94870d0bb33a71da00e41164b403"_. You should find an email about not having any node in the cluster with sufficient memory to run instaGRAAL (it requires a GPU). This was also mentioned by @sanjaysrikakulam [here](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/799#pullrequestreview-1479279997).

Here I see that we have a flavor with a GPU and sufficient memory but it is not referenced in [resources.yaml](https://github.com/usegalaxy-eu/vgcn-infrastructure/blob/2404487d049218f6ec95834e40df2970eae96fb0/resources.yaml). I checked using `openstack flavor list` that the flavor is indeed defined. Should we add it to our inventory?

Reply:
We can only have 2 VMs of that flavor and one of them is already used by Anup. 

I discussed with RZ the possibility of having more resources allocated to the flavor we currently use `g1.c7m20g1` and as I assumed the node (bare metal) where this flavored VM can be created does not have adequate resources.

So the available options are,
1. Move (some) GPUs to other bare metal servers with enough resources
2. Reduce the count of the VMs (from 8 to 4) and double the resources (This also means we will have 2 GPUs attached to every VM and we might be able to configure some tools to use a different GPU but for many, it might not be possible and we might end up with more waiting time and so on. I am not sure whether this setup was ever tested)
3. If we have free RAM slots on this node where this flavor can run we can buy more RAM (RZ needs to check if there are free slots available).

We need to agree on an option and proceed further. To me, option 1 sounds best and probably we will have a short downtime.